src/app/app.component.ts

```diff
@@ -48,12 +48,18 @@ export class AppComponent implements OnDestroy {
         }
       });
 
-    if (this.configService.config?.oidcConfig) {
-      this.oidcService.checkAuth().pipe(
-        filter((d) => d.isAuthenticated),
-        map((d: LoginResponse) => fromAuth.oidcLogin(d)),
-      ).subscribe((action) => this.store.dispatch(action));
-    }
+      const err = new URLSearchParams(window.location.search).get('error');
+      if (err === "interaction_required") {
+        AppComponent.autoLoginFailed = true;
+      } else if (this.configService.config?.oidcConfig && !AppComponent.autoLoginFailed) {
+        this.oidcService.checkAuth().subscribe((loginResponse) => {
+          if (loginResponse.isAuthenticated) {
+            this.store.dispatch(fromAuth.oidcLogin(loginResponse));
+          } else if (!AppComponent.autoLoginFailed) {
+            this.oidcService.authorize(undefined, {customParams:{prompt: 'none'}});
+          }
```

Comment:
If this subscription is only one shot you can free memory with:
```suggestion
        const subscription = this.oidcService.checkAuth().subscribe((loginResponse) => {
          if (loginResponse.isAuthenticated) {
            this.store.dispatch(fromAuth.oidcLogin(loginResponse));
          } else if (!AppComponent.autoLoginFailed) {
            this.oidcService.authorize(undefined, {customParams:{prompt: 'none'}});
          }
          subscription.unsubscribe();
```

Reply:
Added unsubscribe. Unfortunately, had to do it as 
```
let  sub = new Subscription();
sub = subscribe(() => {... sub.unsubscribe(); });
```
to avoid "[Why am I getting a "Cannot access 'variable' before initialization" error from a subscriber referencing itself?](https://stackoverflow.com/questions/58486450/why-am-i-getting-a-cannot-access-variable-before-initialization-error-from-a)"